node_modules/col-core/lib/db.js

```diff
@@ -231,19 +231,31 @@ var setUpModel = function(sequelize) {
     },
     'assetlibrary_url': {
       'type': Sequelize.STRING,
-      'allowNull': true
+      'allowNull': true,
+      'set': function(url) {
+        this.setDataValue('assetlibrary_url', url && url.split('?')[0]);
+      }
     },
     'dashboard_url': {
       'type': Sequelize.STRING,
-      'allowNull': true
+      'allowNull': true,
+      'set': function(url) {
+        this.setDataValue('dashboard_url', url && url.split('?')[0]);
+      }
     },
     'engagementindex_url': {
       'type': Sequelize.STRING,
-      'allowNull': true
+      'allowNull': true,
+      'set': function(url) {
+        this.setDataValue('engagementindex_url', url && url.split('?')[0]);
+      }
     },
     'whiteboards_url': {
       'type': Sequelize.STRING,
-      'allowNull': true
+      'allowNull': true,
+      'set': function(url) {
+        this.setDataValue('whiteboards_url', url && url.split('?')[0]);
+      }
```

Comment:
@paulkerschen The secret sauce is `this.setDataValue()` per Sequelize docs. In my local env, the code above worked as expected.